src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -1,71 +1,92 @@
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'OSX_Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'OSX_Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
-  <ItemGroup>
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\Crc32Helper.cs" />
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipArchive.cs" />
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipArchiveEntry.cs" />
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipArchiveMode.cs" />
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipBlocks.cs" />
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipCustomStreams.cs" />
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipHelper.cs" />
-    <Compile Include="$(SharedOpenSourcePath)System\IO\Compression\ZipVersionNeededValues.cs" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'net46'">
-    <Compile Include="System\IO\Compression\CompressionLevel.cs" />
-    <Compile Include="System\IO\Compression\CompressionMode.cs" />
-    <Compile Include="System\IO\Compression\Deflater.cs" />
-    <Compile Include="System\IO\Compression\DeflateStream.cs" />
-    <Compile Include="System\IO\Compression\GZipStream.cs" />
-    <Compile Include="System\IO\Compression\Inflater.cs" />
-    <Compile Include="System\IO\Compression\ZLibException.cs" />
-    <Compile Include="System\IO\Compression\ZLibNative.cs" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(TargetGroup)' != 'net46'">
-    <Compile Include="System\IO\Compression\ZLibNative.Windows.cs" />
-    <Compile Include="Interop\Interop.zlib.Windows.cs" />
-    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
-      <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
-    <Compile Include="System\IO\Compression\ZLibNative.Unix.cs" />
-    <Compile Include="Interop\Interop.zlib.Unix.cs" />
-    <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
-      <Link>Common\Interop\Unix\Interop.Libraries.cs</Link>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
-    <TargetingPackReference Include="mscorlib" />
-    <TargetingPackReference Include="System" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
-  </ItemGroup>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+    <PropertyGroup>
```

Comment:
Looks like you converted the tabs to 4 spaces for all other projects and xml files in our repo it is 2 spaces can you please undo this?